src/bridge.ts

```diff
@@ -828,7 +825,7 @@ export class Bridge {
             this.addAppServicePath({
                 method: "GET",
                 path: "/_matrix/app/:version(v1|unstable)/thirdparty/protocol/:protocol",
-                checkToken: this.opts.authenticateThirdpartyEndpoints,
+                checkToken: true,
```

Comment:
Nitpick: unless there's a good reason for this to specifically set `checkToken`, just get rid of it, like all of the other calls to `this.addAppServicePath` did.